editor/components/rich-text/tinymce.js

```diff
@@ -96,6 +182,23 @@ export default class TinyMCE extends Component {
 		} );
 	}
 
+	saveEditorNode( editorNode ) {
```

Comment:
Minor: `bindEditorNode` is more consistent with how we name these sorts of functions elsewhere in the codebase.